src/common/playlist/playlist-manager.js

```diff
@@ -41,7 +41,7 @@ class PlaylistManager {
    */
   configure(config: ?KPPlaylistObject, entryList: ?ProviderEntryListObject) {
     if (config) {
-      this._playlist.configure(config);
+      this._playlist.configure(config, Utils.Object.getPropertyPath(this._player.config, 'sources.options'));
```

Comment:
doesn't it enough to send the config? `sources` already exist under config

Reply:
it's complicated that there are multiple sources from a different components

Reply:
`config` it's the playlist config (id, poster etc.) it's not including the item sources